ui/src/components/easter/invitation/InvitationContainer.js

```diff
@@ -1,13 +1,63 @@
-import { Text } from "@chakra-ui/react";
+import * as React from 'react';
+import { useMediaQuery } from '@chakra-ui/media-query';
+import { Center, Image, Stack, Text, Flex, Link, extendTheme } from '@chakra-ui/react';
+import Invitation1 from './blie.png';
```

Comment:
can you move these images to the easter image folder? (hmcchk-web/ui/public/images/easter/)

you can then get the image by pointing to it
reference: https://github.com/hmcc-global/hmcchk-web/blob/main/ui/src/components/home/AboutSection.js#L32